### ISSUE TITLE
User/Developer Manual

### ISSUE THREAD

**Issue by swflint** [zkry/p-search#54]
P-Search is complex enough that it needs a manual.

How can I help?  What sort of docs were you planning on providing?

**Comment by zkry** [comment]
Yes, this is probably the biggest missing part now. My idea was write up a Texinfo document and have it take inspiration from the other Texinfo documentation out there, like org-mode or magit.  This would also allow for a HTML page export which I plan to host at p-search.org. 

Besides a few odds and ends, writing the documentation will be my main focus, which I hope to get written before new years.

When it's written, reading through it and pointing out things that don't make sense or are unclear, or anything else missing would be very helpful. I'll post updates on the docs here.

**Comment by swflint** [comment]
I am very much excited for this!

**Comment by zkry** [comment]
It's very close to done! You can see what exists of it now at https://p-search.org/documentation/

I want to get in a sort of cookbook style section walking through examples of making various extensions.

**Comment by zkry** [comment]
Ok, so I'll consider the documentation done, but I expect there to be a lot of improvements that could be made. If you come across something unclear or missing definitely open up an issue and I can fix it. I got in some examples of making candidate generators. I'll work on the remaining sections later. In the meantime I want to focus on finishing of the search fields work we were discussing.
